chaos_kitten/brain/attack_chainer.py

```
@@ -0,0 +1,236 @@
+        
+        for i, ep in enumerate(self.endpoints):
+            # Find consumed fields
+            for param in ep.get("parameters", []):
+                name = param.get("name")
+                if name:
+                    consumers[name.lower()].append(i)
+            
+            req_body = ep.get("requestBody", {})
+            if req_body:
+                content = req_body.get("content", {})
+                for media_type, media_obj in content.items():
```

Ruff (0.15.1)

[warning] Loop control variable `media_type` not used within loop body

Rename unused `media_type` to `_media_type`

(B007)